tests/integration/external/guzzle6/test_bad_response_exception_async.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+Test that the span for an async external reqest that ends up throwing BadResponseException exception
```

Comment:
copy/paste comment for all tests: `reqest` should be `request`

Reply:
Addressed in 13db0365f685a2a186d570757fee9dc15a7fdfd8.